diracx-core/src/diracx/core/config/__init__.py

```diff
@@ -85,32 +89,21 @@ def clear_caches(self):  # noqa
         pass
 
 
-class LocalGitConfigSource(ConfigSource):
-    scheme = "git+file"
+class BaseGitConfigSource(ConfigSource):
+    scheme = "git"
 
     def __init__(self, *, backend_url: ConfigSourceUrl) -> None:
-        if not backend_url.path:
-            raise ValueError("Empty path for LocalGitConfigSource")
-
-        repo_location = Path(backend_url.path)
-        self.repo_location = repo_location
-        self.repo = git.Repo(repo_location)
+        super().__init__(backend_url=backend_url)
+        self.repo = git.Repo()
```

Comment:
this cannot be in the base

Reply:
Yes, that's what I saw... I correct that